pkg/cmd/cli/sa/new.go

```diff
@@ -0,0 +1,111 @@
+package sa
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/unversioned"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const (
+	NewServiceAccountRecommendedName = "new"
+
+	newServiceAccountShort = `Create a new service account.`
+
+	newServiceAccountLong = `
+Create a new service account.
+
+Service accounts allow system components to access the API independently with privileged tokens.
+This command allows you to create a new service account for your own use. The only default role
+that will be added to the service account by default is the 'system:image-puller' role, so if 
+other roles are necessary, they need to be added using the following commands:
+  
+  $ %[1]s ROLE serviceaccounts/<service-account-name>
+  $ %[2]s ROLE serviceaccounts/<service-account-name>
+`
+
+	newServiceAccountUsage = `%s NAME`
+
+	newServiceAccountExamples = `  # Create a new service account with the name 'custom-sa'
+  $ %[1]s 'custom-sa'
+`
+)
+
+type NewServiceAccountOptions struct {
+	SAName    string
+	Namespace string
+	SAClient  unversioned.ServiceAccountsInterface
+
+	Out io.Writer
+	Err io.Writer
+}
+
+func NewCommandNewServiceAccount(name, fullname string, f *clientcmd.Factory, out io.Writer, addRoleToUserName, addClusterRoleToUserName string) *cobra.Command {
```

Comment:
I don't want this command here.  Creating a new service account should be `oc create serviceacount`.  Pull this out of here and submit a pull upstream to create a new service account.